### PR TITLE
Update CCE Portal Info

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
https://vajira.max.gov/browse/API-2097
Updated CCE documentation to contain authorization info and page link. Also cleaned up some of the beginning text and added partial success scenarios.

### CODE REVIEW

**Comment by taylorrassmannlibertyits** [comment]
NOTE: Need buy in from Darcy (UX) before merging.

**Comment by taylorrassmannlibertyits** [comment]
> NOTE: Need buy in from Darcy (UX) before merging.

Sent Darcy the output, waiting on response.

File: community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java [hunk 1]
```diff
@@ -31,14 +31,34 @@
...
+                    + "\n\n"
+                    + "## Authorization"
+                    + "\n\n"
+                    + "This API uses the [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html)"
+                    + " standard (OAuth 2) to allow the "
+                    + "person being confirmed to log in and provide digital consent. "
+                    + "API requests are authorized using a Bearer token issued through "
+                    + "an OpenID Connect service. The token should be submitted as an "
+                    + "Authorization header in the form Bearer.\n\n"
+                    + "See our [Authorization Guide](https://developer.va.gov/explore/verification/docs/authorization)"
```
Comment (jblefkove-va) [inline community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java:52]: Should this link to the health API authorization guide? https://developer.va.gov/explore/health/docs/authorization

Comment (taylorrassmannlibertyits) [reply community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java]: Definitely! Link should be updated now.

**Pull request merged**